python/grass/script/core.py

```diff
@@ -1939,7 +1939,7 @@ def _create_location_xy(database, location):
 
         os.chdir(cur_dir)
     except OSError as e:
-        raise ScriptError(repr(e))
+        raise ScriptError(repr(e)) from e
```

Comment:
This basically looks like a pointless try except, as the original Error is raised, so why using try-except here, if not the scripterror simplifies the error message presented to the user...

Reply:
ScriptError seems to be the error which is raised by related functions, so I think is is for consistency.

With that said, OSError seems like a fine exception to be raised from project creation functions (which would remove the need for try-except).